skuba-update/test/os/suse/suse.sh

```diff
@@ -116,6 +116,7 @@ check_kubectl_calls() {
         for i in "$@"; do
             echo "$i" >> /commands.txt
         done
+        command -v diff || install_package diffutils
```

Comment:
actually this fails, as the first argument is the repository name (no idea who came up with this.. )

so it should be


```suggestion
        type -p diff || install_package base diffutils
```